static/js/components/UIPreferences.jsx

```diff
@@ -77,6 +85,14 @@ const UIPreferences = () => {
     />
   );
 
+  const useRefMagSwitch = (
+    <Switch
+      value="Plot Total Magnitude / Flux (including reference)"
+      checked={useRefMag}
+      onChange={useRefMagToggled}
+    />
+  );
+
```

Comment:
This is fine but wouldn't it be even more useful if there was a button in the photometry plot to toggle this on/off?

Reply:
Sure but the bokeh stuff is a nightmare. It's not worth the dev time.

Reply:
OK, I say we go for it as is and fix it if needed later.